suzieq/shared/context.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict, List
+from dataclasses import dataclass, field
+
+
+@dataclass
+class SqContext:
+    '''Context passed between the front-end and back-end'''
+    cfg: Dict = field(default_factory=dict)
+    schemas: Dict = field(default_factory=dict)
+
+    pager: bool = False
+    namespace: str = ''
```

Comment:
It would be nice be consistent with single and double quotes

Reply:
Ack